.github/workflows/next-build-nightly.yml

```diff
@@ -0,0 +1,157 @@
+name: next-build-nightly
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
Core tests run at 3 am, so this should be good.
I don't see an option to use `next` release in smoke test, so I assume it runs only on latest canary release.